src/benchmarklib/benchmark_config.cpp

```diff
@@ -7,7 +7,7 @@ BenchmarkConfig::BenchmarkConfig(const BenchmarkMode benchmark_mode, const bool
                                  const Duration& max_duration, const Duration& warmup_duration, const UseMvcc use_mvcc,
                                  const std::optional<std::string>& output_file_path, const bool enable_scheduler,
                                  const uint32_t cores, const uint32_t clients, const bool enable_visualization,
-                                 const bool cache_binary_tables, std::ostream& out)
+                                 const bool validate, const bool cache_binary_tables, std::ostream& out)
```

Comment:
I just realized it's easy to confuse this with mvcc validation, just looking at the parameter name and the fact that we have a "Validate" operator. validate_against_sqlite or sqlite_validation, maybe?

Reply:
`--verify`?

Reply:
better, for sure. And "--verify-against-sqlite" is pretty long, so "--verify" is probably okay, though I would not have a problem with the longer version either.